k8s/bin/vault-to-sealedsecret.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# vault-to-sealedsecret.sh aat /Users/markry/workspace/hmcts/cnp-flux-config/k8s/aat/pub-cert.pem money-claims-test
+
+env=$1
+cert=$2
+namespace=$3
+secrets=(
+    'cmc|notify-api-key'
+    'cmc|oauth-client-secret'
+    'cmc|citizen-oauth-client-secret'
+    'cmc|anonymous-caseworker-username'
+    'cmc|anonymous-caseworker-password'
+    'cmc|system-update-username'
+    'cmc|system-update-password'
+    'cmc|ccd-importer-username' 
+    'cmc|ccd-importer-password' 
+    'ccd|ccd-api-gateway-oauth2-client-secret'
+    'ccd|ccd-admin-web-oauth2-client-secret'
+    'ccpay|gov-pay-keys-cmc'
+)
+
+for i in "${secrets[@]}"; do
+    vault=`echo "$i" | awk -F'|' '{print $1}'`-$env
```

Comment:
normally you avoid backticks as they have sub weird limitations, sub-shell is safer to use
```suggestion
    vault=$(echo "$i" | awk -F'|' '{print $1}')-$env
```